shotover/src/transforms/tee.rs

```diff
@@ -38,12 +49,21 @@ impl TeeBuilder {
             behavior,
             timeout_micros,
             dropped_messages,
+            switch_port,
         }
     }
 }
 
 impl TransformBuilder for TeeBuilder {
     fn build(&self) -> Transforms {
+        let result_source = Arc::new(AtomicResultSource::new(ResultSource::RegularChain));
+
+        if let Some(switch_port) = self.switch_port {
```

Comment:
I think the task and arc need to be shared between all tee instances from the same builder, like you had before.
Otherwise it wont be able to handle 2 incoming connections at once!